app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class AddressSearchController < ApplicationController
+      include RenderConditionConcern
+
+      check_or_render_not_found -> { InPersonConfig.enabled? }
+
+      def index
+        render json: addresses
+      end
+
+      protected
+
+      def addresses
+        suggestion = geocoder.suggest(permitted_params[:address]).first
+        [geocoder.find_address_candidates(suggestion.magic_key).first]
```

Comment:
the `[arr.first]` seemed goofy to me, not sure if this reads better. Also what happens if `suggest()` returns an empty array? should we just bail with an empty array for that?

```suggestion
        suggestion = geocoder.suggest(permitted_params[:address]).first
        return [] if !suggestion
        geocoder.find_address_candidates(suggestion.magic_key).slice(0, 1)
```

Reply:
that just means more proxied requests, unsure if needed for now, but we can always re-evaluate